tests/test_models/test_sarimax_model.py

```diff
@@ -169,7 +170,7 @@ def test_decomposition_hamiltonian_repr_error(dfs_w_exog, components_method_name
 )
 @pytest.mark.parametrize("trend", (None, "t"))
 def test_components_names(dfs_w_exog, regressors, regressors_components, trend, components_method_name, in_sample):
-    expected_components = regressors_components + ["target_component_sarima"]
+    expected_components = regressors_components + ["target_component_arima"]
```

Comment:
???

Reply:
With new default values we by default doesn't have seasonality (because the last component of `seasonal_order` is equal to 0). And in this case we name component `arima` instead of `sarima`.